site/gatsby-site/cypress/integration/downloadIndex.js

```diff
@@ -0,0 +1,92 @@
+import { gql } from '@apollo/client';
+import path from 'path';
+
+describe('Download Algolia Index', () => {
+  const url = '/downloadIndex';
+
+  const downloadsFolder = Cypress.config('downloadsFolder');
+
+  it('Should return a properly constructed Algolia index', () => {
+    cy.visit(url);
+
+    cy.query({
+      query: gql`
+        query {
+          reports(limit: 9999) {
+            _id
+            cloudinary_id
+            date_downloaded
+            date_modified
+            date_published
+            date_submitted
+            description
+            epoch_date_downloaded
+            epoch_date_modified
+            epoch_date_published
+            epoch_date_submitted
+            epoch_incident_date
+            flag
+            image_url
+            incident_date
```

Comment:
incident_date is still on the reports collection? It should be joined in to the rows of the reports, but it should not be in the same collection as reports post-normalization.

Reply:
Yeah, I was thinking that it could be done in another PR since it should also use MongoDB's Date data type,  and that would require more changes yet. But if you think it's better to do it all at once I can add that change here too.